app/controllers/decidim_controller.rb

```diff
@@ -1,3 +1,10 @@
 # frozen_string_literal: true
 class DecidimController < ActionController::Base
+  before_action :set_raven_context
+
+  def set_raven_context
+    return unless Rails.env.production?
+    Raven.user_context({ id: session[:current_user_id] }.merge(session))
```

Comment:
If it's nil, then you'll know there's no user logged in :/ I don't really see the problem here :/